jib-cli/src/main/java/com/google/cloud/tools/jib/cli/cli2/JibCli.java

```diff
@@ -354,4 +380,15 @@ public static void main(String[] args) {
     int exitCode = new CommandLine(new JibCli()).execute(args);
     System.exit(exitCode);
   }
+
+  /** Validater for parameters defined this this class that could not be done declaratively. */
+  public void validate() {
+    if (targetImage.startsWith(TAR_IMAGE_PREFIX)) {
+      if (name == null) {
```

Comment:
we can combine these two conditions?

Reply:
this code is gone, should be in the other PR